docs/bin/manifest.json

```diff
@@ -29,11 +29,6 @@
         "parent": null,
         "markdown_source": "https://github.com/wordpress/secure-custom-fields/blob/trunk/docs/welcome/index.md"
     },
-    "code-reference/abilities": {
-        "slug": "abilities",
-        "parent": "code-reference",
-        "markdown_source": "https://github.com/wordpress/secure-custom-fields/blob/trunk/docs/code-reference/abilities/index.md"
-    },
```

Comment:
@priethor can you please check why is removing the abilities code reference on docs?

Could it be due to refactors?

Reply:
Yes. I didn't change anything manually :)